### PR TITLE
KOGITO-8165 Correct Build Chain jenkinsfile

### PR DESCRIPTION
**Pull request by radtriste**
https://issues.redhat.com/browse/KOGITO-8165

### CODE REVIEW

File: .ci/jenkins/Jenkinsfile.buildchain [hunk 1]
```diff
@@ -76,12 +76,19 @@ pipeline {
             }
             steps {
                 script {
-                    env.BUILD_MVN_OPTS_CURRENT = getBuildMavenOptsCurrent()
-                    echo "BUILD_MVN_OPTS_CURRENT = ${BUILD_MVN_OPTS_CURRENT}"
+                    env.BUILD_MVN_OPTS_CURRENT = env.BUILD_MVN_OPTS_CURRENT ?: ''
+                    env.BUILD_MVN_OPTS = env.BUILD_MVN_OPTS ?: ''
+                    
+                    env.BUILD_MVN_OPTS = "${env.BUILD_MVN_OPTS} -Dmaven.wagon.http.ssl.insecure=true -Dmaven.test.failure.ignore=true"
```
Comment (Ginxo) [inline .ci/jenkins/Jenkinsfile.buildchain]:
```suggestion
                    env.BUILD_MVN_OPTS = "${env.BUILD_MVN_OPTS ?: ''} -Dmaven.wagon.http.ssl.insecure=true -Dmaven.test.failure.ignore=true"
```

File: .ci/jenkins/Jenkinsfile.buildchain [hunk 2]
```diff
@@ -76,12 +76,19 @@ pipeline {
...
+                    env.BUILD_MVN_OPTS_CURRENT = "${env.BUILD_MVN_OPTS_CURRENT} ${getBuildMavenOptsCurrent()}"
```
Comment (Ginxo) [inline .ci/jenkins/Jenkinsfile.buildchain]:
```suggestion
                    env.BUILD_MVN_OPTS_CURRENT = "${env.BUILD_MVN_OPTS_CURRENT ?: ''} ${getBuildMavenOptsCurrent()}"
```

File: .ci/jenkins/Jenkinsfile.buildchain [hunk 3]
```diff
@@ -76,12 +76,19 @@ pipeline {
             }
             steps {
                 script {
-                    env.BUILD_MVN_OPTS_CURRENT = getBuildMavenOptsCurrent()
-                    echo "BUILD_MVN_OPTS_CURRENT = ${BUILD_MVN_OPTS_CURRENT}"
+                    env.BUILD_MVN_OPTS_CURRENT = env.BUILD_MVN_OPTS_CURRENT ?: ''
```
Comment (Ginxo) [inline .ci/jenkins/Jenkinsfile.buildchain]:
```suggestion
```

File: .ci/jenkins/Jenkinsfile.buildchain [hunk 4]
```diff
@@ -76,12 +76,19 @@ pipeline {
...
+                    env.BUILD_MVN_OPTS = env.BUILD_MVN_OPTS ?: ''
```
Comment (Ginxo) [inline .ci/jenkins/Jenkinsfile.buildchain]:
```suggestion
```

File: .ci/jenkins/Jenkinsfile.buildchain [hunk 5]
```diff
@@ -76,12 +76,19 @@ pipeline {
...
+
```
Comment (Ginxo) [inline .ci/jenkins/Jenkinsfile.buildchain]:
```suggestion
```

**Review by Ginxo — APPROVED** [review]
LGTM, just check my suggestions

**Comment by radtriste** [comment]
tested successfully in https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-runtimes.tests.downstream.kogito-examples/1455/

**Pull request merged**